08-rails/installing-foundation.md

```diff
@@ -0,0 +1,120 @@
+#  Installing Foundation By Default
+
+## Learning Goals
+- Practice modifying our `.rails-template.rb` file to add an additional gem and content.
+- Learn how to install and use the Foundation Framework via the gem.
+
+## Installing the Foundation Gem
+
+To install Foundation in your Rails project via the Gem first add the following to your Gemfile.
+
+```ruby
+gem 'foundation-rails'
+```
+
+And run `bundle install` from the terminal in your rails project.  
+
+```bash
+$  bundle install
+```
+
+
+Then in your terminal in the rails project folder run the command:
+
+```bash
+$  rails generate foundation:install --force
+```
+
+This will add the following files to your project
+-  app/assets/javascripts/application.js
+-  app/assets/stylesheets/foundation_and_overrides.scss
+-  app/assets/stylesheets/_settings.scss
+-  app/assets/stylesheets/application.css
+-  app/assets/stylesheets/foundation_and_overrides.scss  
+
+It will also overwrite the default layout `app/views/layouts/application.html.erb`
```

Comment:
We should probably provide a warning that any changes they've manually made to the layout template will be removed at the same time.